src/cmr/concepts/revision.js

```diff
@@ -0,0 +1,111 @@
+import {
+  snakeCase,
+  kebabCase,
+  _
+} from 'lodash'
+import { downcaseKeys } from '../../utils/downcaseKeys'
+import Concept from './concept'
+
+export default class Revision extends Concept {
+  /**
+   * Instantiates an Revision object
+   * @param {Object} headers HTTP headers provided by the query
+   * @param {Object} requestInfo Parsed data pertaining to the Graph query
+   * @param {Object} params GraphQL query parameters
+   */
+
+  // eslint-disable-next-line no-useless-constructor
+  constructor(conceptType, headers, requestInfo, params) {
+    // This concept uses the "-" character to delineate spaces in CMR we must pass it
+    // in this form to fetch order option concepts from CMR
+    super(conceptType, headers, requestInfo, params)
+  }
+
+  /**
+   * Parses the response from the umm endpoint
+   * @param {Object} ummResponse HTTP response from the CMR endpoint
+   * @param {Array} ummKeys Array of the keys requested in the query
+   */
+  async parseUmm(ummResponse, ummKeys) {
```

Comment:
Add a comment above describing what the difference in this parseUmm vs the parseUmm in concept.js